### PR TITLE
Update main with changes from develop branch

### PR DESCRIPTION
**Pull request by 2color**

### CODE REVIEW

**Review by lidel — APPROVED** [review]
A bit concerning that so many changes were not shipped to production for over a year, but I eyeballed preview at https://bafybeia5ur3p2wnyomqlmsbhfgo6eymg74gstnphvmy4sabe5awc2ee2je.on.fleek.co/, and it looks fine.


Merging mainly to ship domain change introduced in https://github.com/ipfs/ecosystem-directory/pull/238

**Comment by orvn** [comment]
I looked through the commit list and this was safe to move to production! I think we didn't have approvals on some of the features at the time.

One note: any content changes that were made to project `json` files should be made in the Airtable, otherwise they might get overwritten in the future.

File: content/data/settings.json [hunk 1]
```diff
@@ -33,5 +33,20 @@
     "mediaAltAtts": true,
     "linksAltAtts": true,
     "singularTagLinks": true
-  }
+  },
+  "embeddable_view": {
+    "host": "https://bafybeibjasmf2bvswm3ejg4shppxtw5l36bfn7uouypjm6teqwpcpxi4pi.on.fleek.co/",
```
Comment (lidel) [inline content/data/settings.json:38]: @orvn  I missed this one, do we need to change this to `ecosystem.ipfs.tech`?

Comment (lidel) [reply content/data/settings.json:38]: I'm around if you want to push this over the finish line and add this section to https://ipfs.tech

**Pull request merged**